src/main/java/online/book/store/util/SortUtil.java

```diff
@@ -0,0 +1,12 @@
+package online.book.store.util;
+
+import org.springframework.data.domain.Sort;
+
+public class SortUtil {
+    public static Sort.Order parseSortOrder(String sort) {
```

Comment:
any Util classes should 
1) be final
2) have all static methods
3) not be accessible to instantiate (e.g. private constructor)